database/youtube/web-development.json

```diff
@@ -55,11 +55,20 @@
       "language": "english"
     },
     {
-        "name": "CodeWithHarry",
-        "description": "This channel provides free Web Development course for beginners.",
-        "url": "https://www.youtube.com/@CodeWithHarry/playlists",
+
+      "name": "Thenewboston",
+      "description": "Thenewboston is a great channel for starting you web development journey and other programming languages.",
```

Comment:
Great start @parma1111! 😊 However, Web Devs might ask:

1. What does make this source different from other YouTube channels?

Consider providing this info to help pique these individuals' interest in the source.

My next suggestion is to fix the grammatical error (e.g.,your) so the sentence can be clearer. 

Hope this helps! 😁